docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+version: "3.8"
+
+services:
+  app:
+    container_name: frontend-dev
+    build:
+      context: .
+      target: development
+    volumes:
+      - ./app:/app
+    env_file: ./.env
+    ports:
+      - ${NODE_LOCAL_PORT}:${NODE_DOCKER_PORT}
+    networks:
+      - frontend
+      - backend
+
+  #  flask:
```

Comment:
Don't have code (or in this case config) that is commented out.

Remove these lines since they are not necessary.